index.html

```diff
@@ -1,221 +1,293 @@
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+
+            <ul class="projects">
+              <li class="project-language css">Css</li>
+              <li class="project-language html">Html</li>
+              <li class="project-language bootstrap">Bootstrap</li>
+              <li class="project-language ruby">Ruby</li>
+            </ul>
+
+            <button type="button" class="button">See Project</button>
+          </div>
+
+          <div id="Multi-post-3" class="multi-post">
+            <img src="images/Placeholder.png" alt="" />
+            <h2 class="white-title">Multi-Post Stories</h2>
+            <p class="experience">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+
+            <ul class="projects">
+              <li class="project-language css">Css</li>
+              <li class="project-language html">Html</li>
+              <li class="project-language bootstrap">Bootstrap</li>
+              <li class="project-language ruby">Ruby</li>
+            </ul>
+
+            <button type="button" class="button">See Project</button>
+          </div>
+
+          <div id="Multi-post-4" class="multi-post">
+            <img src="images/Placeholder.png" alt="" />
+            <h2 class="white-title">Multi-Post Stories</h2>
+            <p class="experience">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+
+            <ul class="projects">
+              <li class="project-language css">Css</li>
+              <li class="project-language html">Html</li>
+              <li class="project-language bootstrap">Bootstrap</li>
+              <li class="project-language ruby">Ruby</li>
+            </ul>
+
+            <button type="button" class="button">See Project</button>
+          </div>
+        </section>
+
+        <section class="form">
+          <h2 class="contact-us">CONTACT US</h2>
+          <p class="info">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form id="form" action="/my-handling-form-page" method="post">
+            <ul>
+              <li>
+                <input
+                  type="text"
+                  id="name"
+                  name="user_name"
+                  maxlength="30"
+                  placeholder="Text Input"
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly add `required` attribute to make sure that you can't submit this form with this input-text empty.
- [ ] <b>REQUIRED:</b> Placeholders are for helping users understand what needs to be put in `input-text`, so it is better to be more specific when writing them. For example: above you said `Text input` and it can be anything. which means it will be confusing for the users to understand what to put in there. the correct placeholder value can be like `Enter the name`, `Full name`, `First name and Last name` etc...